### PR TITLE
Header / Footer

### PR DESCRIPTION
**Pull request by kateharwood**
First pass at navbar.

@Letsgohyde still need to add in the real icons (and definitely need a better version of the logo, I'm just using the favicon currently)

Also, does not play well with mobile / is not responsive yet. What are our plans regarding mobile and responsiveness for this iteration? Do we have separate designs?

Screen Shot 2020-11-12 at 2 11 51 PM

### CODE REVIEW

**Comment by kateharwood** [comment]
Added footer

Screen Shot 2020-11-13 at 2 32 20 PM

**Comment by kateharwood** [comment]
Header updated: 
Screen Shot 2020-11-13 at 4 49 53 PM

File: static/font-awesome/api.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 384 512"><!-- Font Awesome Free 5.15.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) --><path d="M384 144c0-44.2-35.8-80-80-80s-80 35.8-80 80c0 36.4 24.3 67.1 57.5 76.8-.6 16.1-4.2 28.5-11 36.9-15.4 19.2-49.3 22.4-85.2 25.7-28.2 2.6-57.4 5.4-81.3 16.9v-144c32.5-10.2 56-40.5 56-76.3 0-44.2-35.8-80-80-80S0 35.8 0 80c0 35.8 23.5 66.1 56 76.3v199.3C23.5 365.9 0 396.2 0 432c0 44.2 35.8 80 80 80s80-35.8 80-80c0-34-21.2-63.1-51.2-74.6 3.1-5.2 7.8-9.8 14.9-13.4 16.2-8.2 40.4-10.4 66.1-12.8 42.2-3.9 90-8.4 118.2-43.4 14-17.4 21.1-39.8 21.6-67.9 31.6-10.8 54.4-40.7 54.4-75.9zM80 64c8.8 0 16 7.2 16 16s-7.2 16-16 16-16-7.2-16-16 7.2-16 16-16zm0 384c-8.8 0-16-7.2-16-16s7.2-16 16-16 16 7.2 16 16-7.2 16-16 16zm224-320c8.8 0 16 7.2 16 16s-7.2 16-16 16-16-7.2-16-16 7.2-16 16-16z"/></svg>
```
Comment (tildechris) [inline static/font-awesome/api.svg:1]: Instead of checking these in, can you pull them down from npm?

**Review by tildechris — CHANGES_REQUESTED** [review]

File: static/font-awesome/api.svg (hunk 1 above)
Comment (tildechris) [reply static/font-awesome/api.svg:1]: Let's handle this in another PR.

**Review by tildechris — APPROVED** [review]
LGTM, thanks Kate!

File: config.toml [hunk 2]
```diff
@@ -8,27 +8,37 @@ relativeURLs = false
 
 [permalinks]
     post = "/:year/:month/:day/:slug/"
-
 [[menu.main]]
-    identifier = "main"
-    name = "Delphi"
+    identifier = "about"
+    name = "About"
     url = "https://delphi.cmu.edu/"
```
Comment (sgratzl) [inline config.toml:14]: please use a relative link here `/about` isn't it?

File: config.toml [hunk 3]
```diff
@@ -8,27 +8,37 @@ relativeURLs = false
...
+[[menu.main]]
+    identifier = "team"
+    parent = "about"
+    name = "Team"
+    url = "#"
     weight = 1
 [[menu.main]]
     name = "COVIDcast"
+    pre = "map"
     url = "https://covidcast.cmu.edu"
```
Comment (sgratzl) [inline config.toml:26]: will soon be `/covidcast`

File: config.toml [hunk 4]
```diff
@@ -8,27 +8,37 @@ relativeURLs = false
...
     weight = 2
-[[menu.main]]
-    name = "Data"
-    url = "https://cmu-delphi.github.io/delphi-epidata/"
-    weight = 3
 [[menu.main]]
     name = "Blog"
+    pre = "blog"
     url = "https://delphi.cmu.edu/blog"
```
Comment (sgratzl) [inline config.toml:31]: same here `/blog`

File: themes/delphi/layouts/partials/delphi-text-logo.html [hunk 5]
```diff
@@ -0,0 +1 @@
+<a class="delphi-text-logo" href="https://delphi.cmu.edu/">DELPHI</a>
```
Comment (sgratzl) [inline themes/delphi/layouts/partials/delphi-text-logo.html:1]: see https://gohugo.io/functions/relref/ to create references in hugo

File: themes/delphi/layouts/partials/footer/desktop.html [hunk 6]
```diff
@@ -0,0 +1,40 @@
+<div class="uk-visible@s footer">
+    <table class="uk-table footer-table">
+        <thead>
+            <tr>
+                <th class="header">
```
Comment (sgratzl) [inline themes/delphi/layouts/partials/footer/desktop.html:5]: should we prefix css classnames by their area?, e.g. `footer-header`?

File: themes/delphi/layouts/partials/head.html [hunk 7]
```diff
@@ -13,6 +13,12 @@
     <!-- <link rel="mask-icon" href="{{ "safari-pinned-tab.svg" | relURL }}" color="#FF3DB4"> -->
     <meta name="theme-color" content="#ffffff">
 
+    <!-- Open Sans Font -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@600&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@700&display=swap" rel="stylesheet">
```
Comment (sgratzl) [inline themes/delphi/layouts/partials/head.html:20]: is it possible to load all fonts at one?

File: themes/delphi/layouts/partials/nav.html [hunk 8]
```diff
@@ -1,9 +1,58 @@
...
       <ul class="uk-navbar-nav">
-          {{ range .Site.Menus.main.ByWeight }}
-          <li class="{{ if eq .Identifier "main" }}uk-active{{ end }}"><a href="{{ .URL | relLangURL }}">{{ .Name }}</a></li>
-          {{ end }}
+        <li>
+          {{partial "delphi-text-logo.html"}}
+        </li>
+        {{ $currentPage := . }}
+        {{ range .Site.Menus.main.ByWeight }}
+        <li class="{{ if (eq $currentPage.URL .URL) }}active{{ end }}">
```
Comment (sgratzl) [inline themes/delphi/layouts/partials/nav.html:9]: have you tried using https://gohugo.io/functions/hasmenucurrent/? instead

**Review by sgratzl — COMMENTED** [review]
can you resolve the conflicts, please

Moreover, can you use relative links otherwise it won't work on our test environment

File: themes/delphi/layouts/partials/nav.html (hunk 8 above)
Comment (kateharwood) [reply themes/delphi/layouts/partials/nav.html:9]: Yes, that wasn't working for me, and I looked at some other posts and saw that it seems not to work without putting the menu items in the "Front Matter" for the pages, and I saw this used as a workaround

File: config.toml (hunk 2 above)
Comment (kateharwood) [reply config.toml:14]: Hmm I'm not sure if matching the active url to these urls to highlight active tab will work if these are changed to be relative, might have to rethink that then if it's important to make these relative.

File: themes/delphi/layouts/partials/head.html (hunk 7 above)
Comment (kateharwood) [reply themes/delphi/layouts/partials/head.html:20]: Will do

File: themes/delphi/layouts/partials/footer/desktop.html (hunk 6 above)
Comment (kateharwood) [reply themes/delphi/layouts/partials/footer/desktop.html:5]: Will do

File: themes/delphi/layouts/partials/delphi-text-logo.html (hunk 5 above)
Comment (kateharwood) [reply themes/delphi/layouts/partials/delphi-text-logo.html:1]: How do you use this to link to the homepage from anywhere on the site? Relref seems to be relative to where the user is on the site? Definitely not understanding something here.

File: config.toml (hunk 2 above)
Comment (kateharwood) [reply config.toml:14]: Actually I take that back, it looks like it works.

**Review by tildechris — APPROVED** [review]
LGTM.

**Pull request merged**